test/verify/check-system-info

```diff
@@ -421,6 +430,85 @@ class TestSystemInfo(MachineCase):
 
         self.machine.execute("umount /sys/class/dmi")
 
+    @skipImage("Added in 190", "rhel-7-6-distropkg")
```

Comment:
Please refer to the PR# instead; easier to find and this PR isn't guaranteed to land by Wednesday